### PR TITLE
First draft of Romans 12 blog post

### PR DESCRIPTION
**Pull request by andygeers**
What do you reckon, James?

### CODE REVIEW

File: src/blog/2019/one-body-many-members.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: One Body, Many Members
+tags: Build
+author: Andy Geers
+date: 2019-10-01
```
Comment (jamesdoc) [inline src/blog/2019/one-body-many-members.md]: Date should be later on… will publish this Thursday next week?

File: src/blog/2019/one-body-many-members.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+intro:
```
Comment (jamesdoc) [inline src/blog/2019/one-body-many-members.md]: This is the two sentances that appears on the blog listing page…

File: src/blog/2019/one-body-many-members.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## 1. Within Kingdom Code we are one body with many members
+
+One of the glorious things about Kingdom Code is the range of different skills represented within the community. Something we often hear when people find out about Kingdom Code is “Oh, but I’m not a coder…” - but praise God that we’re not all coders! A good hackathon team benefits massively from a range of input, from designers to project managers… and yes, ultimately they usually need coders too!
```
Comment (jamesdoc) [inline src/blog/2019/one-body-many-members.md]: Can you pull this out a little bit wider than a hackathon context for this blog post? Perhaps something along the lines of 'a good product team'?

**Comment by andygeers** [comment]
@jamesdoc Just pushed some changes - hopefully that does the job!

**Pull request merged**